PKGBUILD

```diff
@@ -1,62 +1,41 @@
 # Maintainer: Kris Nóva <kris@nivenly.com>
-pkgname="falco"
-pkgver="0.29.1"
+
+pkgname=falco-bin
+pkgver=0.29.1
 pkgrel=1
 pkgdesc="Falco Runtime Security"
-arch=("x86_64")
-url="https://github.com/kris-nova/aur-falco"
-license=("Apache2")
-groups=()
-depends=("linux-headers")
+arch=(x86_64)
+url="https://falco.org/"
+license=(Apache)
+depends=(dkms linux-headers)
 makedepends=()
-checkdepends=()
-optdepends=()
-provides=()
-conflicts=()
-replaces=()
+provides=(falco)
+conflicts=(falco)
 backup=()
-options=()
-install=
-changelog=
+source_x86_64=("https://download.falco.org/packages/bin/${arch}/falco-${pkgver}-x86_64.tar.gz")
```

Comment:
How do you guarantee that the precompiled binary by falco works with the Arch Linux kernel?